.github/issue_template.md

```diff
@@ -0,0 +1,9 @@
+Please be sure to include:
+
+* what operating system you are developing on (linux or macOS)
+
+* for macOS, what version of Xcode you are using (`xcodebuild -version`)
+
+* what branch/tag of SwiftProtobuf you are using (1.0.0, etc.)
+
+* what version of Swift is your code set to compile with
```

Comment:
I'm not sure if it's worth specifically asking people to try to:
 * copy/paste the specific compiler error or warning they're concerned about
 * Include a short piece of code that demonstrates a problem they're seeing

Reply:
Added some more.